<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMachineDimension</class>
 <widget class="QWidget" name="SMachineDimension">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>679</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="img_machine">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../Resources/pyleecan.qrc">:/images/images/MachineSetup/MachineDimension/Dimension_Shaft_Rotor_Stator.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>270</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>268</width>
          <height>616</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="g_stator">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Stator</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="in_SRext">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Stator external radius</string>
              </property>
              <property name="text">
               <string>Rext</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="FloatEdit" name="lf_SRext">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Stator external radius</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="unit_SRext">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="in_SRint">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Stator internal radius</string>
              </property>
              <property name="text">
               <string>Rint</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="FloatEdit" name="lf_SRint">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Stator internal radius</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="unit_SRint">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="g_rotor">
           <property name="title">
            <string>Rotor</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="in_RRext">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Rotor external radius</string>
              </property>
              <property name="text">
               <string>Rext</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="FloatEdit" name="lf_RRext">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Rotor external radius</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="unit_RRext">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="in_RRint">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Rotor internal radius</string>
              </property>
              <property name="text">
               <string>Rint</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="FloatEdit" name="lf_RRint">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Rotor internal radius</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="unit_RRint">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="out_airgap">
           <property name="toolTip">
            <string>Magnetic airgap width (distance between stator bore and rotor bore radii)</string>
           </property>
           <property name="text">
            <string>airgap = </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="g_shaft">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Shaft</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="WMatSelectV" name="w_mat_0" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="in_Lshaft">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Shaft length</string>
                </property>
                <property name="text">
                 <string>Lshaft</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="FloatEdit" name="lf_Lshaft">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Shaft length</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="unit_Lshaft">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>m</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="out_Drsh">
              <property name="toolTip">
               <string>Shaft Diameter</string>
              </property>
              <property name="text">
               <string>Drsh = 2*Rotor.Rint</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="g_frame">
           <property name="title">
            <string>Frame</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="WMatSelectV" name="w_mat_1" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="1">
               <widget class="FloatEdit" name="lf_Lfra">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Frame length</string>
                </property>
                <property name="whatsThis">
                 <string>Frame length</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="in_Lfra">
                <property name="toolTip">
                 <string>Frame length</string>
                </property>
                <property name="whatsThis">
                 <string>Frame length</string>
                </property>
                <property name="text">
                 <string>Lfra</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="FloatEdit" name="lf_Wfra">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Frame width</string>
                </property>
                <property name="whatsThis">
                 <string>Frame width</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="in_Wfra">
                <property name="toolTip">
                 <string>Frame width</string>
                </property>
                <property name="whatsThis">
                 <string>Frame width</string>
                </property>
                <property name="text">
                 <string>Wfra</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="unit_Lfra">
                <property name="toolTip">
                 <string>Frame length</string>
                </property>
                <property name="whatsThis">
                 <string>Frame length</string>
                </property>
                <property name="text">
                 <string>m</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="unit_Wfra">
                <property name="toolTip">
                 <string>Frame width</string>
                </property>
                <property name="whatsThis">
                 <string>Frame width</string>
                </property>
                <property name="text">
                 <string>m</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_previous">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatEdit</class>
   <extends>QLineEdit</extends>
   <header>.....GUI/Tools/FloatEdit.h</header>
  </customwidget>
  <customwidget>
   <class>WMatSelectV</class>
   <extends>QWidget</extends>
   <header>.....GUI.Dialog.DMatLib.WMatSelect.WMatSelectV.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>lf_SRext</tabstop>
  <tabstop>lf_SRint</tabstop>
  <tabstop>lf_RRext</tabstop>
  <tabstop>lf_RRint</tabstop>
  <tabstop>g_shaft</tabstop>
  <tabstop>lf_Lshaft</tabstop>
  <tabstop>g_frame</tabstop>
  <tabstop>lf_Wfra</tabstop>
  <tabstop>lf_Lfra</tabstop>
  <tabstop>b_previous</tabstop>
  <tabstop>b_next</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Resources/pyleecan.qrc"/>
  <include location="../../../Resources/pyleecan.qrc"/>
  <include location="../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
